shared/stats/rollup.config.mjs

```diff
@@ -0,0 +1,36 @@
+import { join } from 'path'
+
+import configESM from 'govuk-frontend/rollup.esm.config.mjs'
+import configUMD from 'govuk-frontend/rollup.umd.config.mjs'
+import { paths } from 'govuk-frontend-config'
+import { visualizer } from 'rollup-plugin-visualizer'
+
+/**
+ * Rollup config with stats output
+ *
+ * @param {Record<string, string | undefined>} args - Rollup CLI arguments
+ * @returns {import('rollup').MergedRollupOptions[]} Rollup options
+ */
+export default ({ i: input }) => [
+  configESM({
+    i: input,
+    d: join(paths.stats, 'dist/report/esm')
+  }),
+  configUMD({
+    i: input,
+    d: join(paths.stats, 'dist/report/umd')
+  })
```

Comment:
How do the stats differ from the ESM build? The modules should be the same, with the same code built in and the only overhead being the UMD boilerplate, which will likely get dropped for v5.

Reply:
The UMD build is hugely different as it's an all-in-one bundle

I want to replace it with an ESM bundle though (rather than our default `{ preserveModules: true }`)

Reply:
Each file is a custom bundle that can be imported standalone

Versus modules where import/require resolution is still needed

Reply:
There's something I must be misunderstanding there. Let's have a call tomorrow to clarify 😊

Reply:
Yeah sounds good 😊

I'll add some background reading, compare these two:

```
scripts.compile('**/!(*.test).mjs', options) // compile:js
scripts.compile('!(*.test).mjs', options) // compile:mjs
```

See the globs?

For `compile:js` Rollup processes every file (recursively) into a separate UMD bundle
For `compile:mjs` Rollup processes only the entry points into ES modules

Which is why when you import multiple UMD files you get the polyfills multiple times

```js
require('govuk-frontend/dist/govuk/components/accordion/accordion')
require('govuk-frontend/dist/govuk/components/character-count/character-count')
require('govuk-frontend/dist/govuk/components/character-count/checkboxes')
```

If you change the format to `cjs` and enable `{ preserveModules }` you'll see the difference

Reply:
✅ UMD stats build has now been removed